doc/api/process.md

```diff
@@ -2305,6 +2305,222 @@ import { loadEnvFile } from 'node:process';
 loadEnvFile();
 ```
 
+### Environment variables file parser specification
+
+This section describes how the environment variables file parser reads a file
+and sets up the environment variables in Node.js.
+
+1. **Basic parsing**:
```

Comment:
The docs generally avoid bold. Rather than this being a numbered list you could make subheadings:

```suggestion
#### Basic parsing
```